docs/production-deployment/cloud/billing-and-cost.mdx

```diff
@@ -0,0 +1,77 @@
+---
+id: billing-and-cost
+title: Billing and Cost
+sidebar_label: Billing and cost
+sidebar_position: 3
+description: As an Account Owner, you can access and manage billing details anytime, with invoices available for download on the Billing page. The typical billing cycle begins on the first of the month (UTC).
+slug: /cloud/billing-and-cost
+toc_max_heading_level: 4
+keywords:
+  - billing
+  - invoices
+  - cost management
+  - account owner
+  - billing cycle
+  - payment options
+  - credit usage
+  - temporal cloud
+  - account balance
+  - invoice history
+tags:
+  - billing
+  - invoices
+  - cost management
+  - account owner
+  - billing cycle
+  - payment options
+  - credit usage
+  - temporal cloud
+  - account balance
+  - invoice history
+---
+
+The Account Owner holds the ability to access and view detailed billing and cost information at any time, ensuring full transparency and control over financial aspects. If you need to assess your spending patterns, check invoice histories, or handle payment details, Temporal Cloud's Cost and Billing capabilities are able to provide the information you need.
+
+## About the Billing page
+
+The **Billing** page includes the following information:
+
+- Your current balance
+- Recent bill
+- Invoice history
+- (If applicable) Credit Grants
+- (If applicable) Credit Purchase
+- (If applicable) Credit Usage
+- (If applicable) Credits Remaining
+
+If you are not on a standard plan, the Billing page may not include all of the information stated above. 
+
+## How to navigate to your invoices
+
+To navigate to your invoices, go to **Settings > Billing**
```

Comment:
I would either add more text here or remove the header since it's looking really short. I assume the steps are something like this. 
"To review your invoices, follow these steps:

1. Navigate to the **Settings** tab from your dashboard.
2. Select **Billing** from the menu options.
3. In the **Invoices** section, locate and download the invoice you wish to review by clicking the download icon next to the invoice date."